src/applications/vaos/containers/ReviewPage.jsx

```diff
@@ -65,10 +69,43 @@ export class ReviewPage extends React.Component {
           </LoadingButton>
         </div>
         {submitStatus === FETCH_STATUS.failed && (
-          <AlertBox status="error" headline="We're sorry. Something went wrong">
-            We ran into a problem trying to submit your request. Please try
-            again later.
-          </AlertBox>
+          <AlertBox
+            status="error"
+            headline={`Your ${
+              isDirectSchedule ? 'appointment' : 'request'
+            } didn’t go through`}
+            content={
```

Comment:
There's a bug in AlertBox that means we have to use `content` if the content is going to change.